### PR TITLE
Add constants for get_user_origin()

### PR DESCRIPTION
**Pull request by justgo97**
Are the names okay for this constants ?

### CODE REVIEW

**Comment by OciXCrom** [comment]
In my opinion it's better to add the constant names in the description as well rather than saying to seacrh for them in another file - having to do this every time is frustrating.

**Comment by Arkshine** [comment]
@justgo97 are you going to do OciXCrom's suggestion?

File: plugins/include/amxconst.inc [hunk 1]
```diff
@@ -533,4 +533,16 @@ enum
...
+/**
+ *  Constants for get_user_origin()
+ */
+enum
+{
+	Origin_Client = 0, 	// Client's Origin
+	Origin_Eyes, 		// Eyes (and Weapon) Origin
+	Origin_AimEndClient, 	// Aim End Origin from Client's Position
+	Origin_AimEndEyes, 	// Aim End Origin from Eyes Position
+	Origin_CSLastBullet 	// Last Bullet's Origin (Counter-Strike)
```
[the remark this answers is not in the document]
Comment (OciXCrom) [reply plugins/include/amxconst.inc]: The second one sounds better.

**Comment by justgo97** [comment]
I no longer have access for this branch, i created a new pull request here https://github.com/alliedmodders/amxmodx/pull/604 and took all suggestions into consideration if any thing else needs to be change, let me know

**Pull request closed without merging**